notebook/before-notebook.d/10_workdir.sh

```diff
@@ -3,6 +3,7 @@ set -x
 workdir=/work
 expdir=/exp
 archivedir=/tmp/_archive
+gittempdir=/tmp/_git_artifact
```

Comment:
Any reason why gittempdir can't just be $archivedir (or just use archivedir instead of gittempdir in the following code?)